docs/installation.md

```diff
@@ -155,22 +155,14 @@ Then create the database, associated user, and schema for that user:
 
 ```bash
 dropdb --if-exists cfgov && dropuser --if-exists cfpb
-createuser cfpb && createdb -O cfpb cfgov
+createuser --createdb cfpb && createdb -O cfpb cfgov
```

Comment:
If we were to set a password for the "cfpb" user, then both local Postgres and Docker Postgres will have the same URL (Docker currently has `cfpb` as the password, local has no password).